dod-advana/gamechanger-web
# UOT-128830 - UX Fixes for Covid-19 Clone

## Description

1. Show the number of search results
2. Update the logo used in title bar and side menu
3. Change the "more" link color on cards to match the “open” link color
4. Move the grid filter, export, copy link button to the right (already was done)

"Before" pics:
<img width="379" alt="Screen Shot 2022-02-24 at 3 19 23 PM" src="https://user-images.githubusercontent.com/98841329/155624575-d1aa4191-5314-4618-bf5f-dd728d8afedf.png">
<img width="1630" alt="Screen Shot 2022-02-24 at 3 19 10 PM" src="https://user-images.githubusercontent.com/98841329/155624598-eabe7cc0-24ff-4aec-8e73-8a402464a665.png">

## !vibez

Phew, my first deep dive into clones!

## Related Issue/Ticket

[UOT-128830](https://jira.di2e.net/browse/UOT-128830)

Note that the fix to show the number of search results will also affect other clones that use the defaultTitleBarHandler and simpleSearchHandler, which is a small part of tickets [UOT-128829](https://jira.di2e.net/browse/UOT-128829) and [UOT-128831](https://jira.di2e.net/browse/UOT-128831)

## Types of changes

## Smoke testing instructions

1. Verify that basic UX changes have been enacted on COVID-19 clone (link color/logo/positioning of grid filter)
2. Test that when viewing search results, COVID-19 clone now includes a bar at the top that tells you the total number of results
3. Verify that this number of results is the total number of results, not the number currently visible on that page
4. Check that this is also the case for other clones that use the defaultTitleBarHandler and simpleSearchHandler

"After" pics:
<img width="379" alt="Screen Shot 2022-02-24 at 3 19 55 PM" src="https://user-images.githubusercontent.com/98841329/155625181-85798a41-20ce-453e-b363-e35261cdced9.png">
<img width="1630" alt="Screen Shot 2022-02-24 at 3 18 39 PM" src="https://user-images.githubusercontent.com/98841329/155625187-e80c6708-c25f-40d1-bcb3-3901599519cd.png">

## Checklist:
